### ISSUE TITLE
RegionFixer is not Python 3.10 compatible due to use of deprecated import in nbt.py

### ISSUE THREAD

**Issue by sebkuip** [Fenixin/Minecraft-Region-Fixer#164]
In the nbt.py file, the following import was done
```from collections import MutableMapping, MutableSequence, Sequence```

The importing of abc's from the collections library has been deprecated in python for a while. And has now been finally removed in python 3.10
https://bugs.python.org/issue25988
https://bugs.python.org/issue36952
```
\Minecraft-Region-Fixer-0.3.3>python regionfixer.py --help
Traceback (most recent call last):
    File "C:\Users\Sebastian Kuipers\Downloads\Minecraft-Region-Fixer-0.3.3\regionfixer.py", line 32, in <module>
        from regionfixer_core.interactive import InteractiveLoop
    File "C:\Users\Sebastian Kuipers\Downloads\Minecraft-Region-Fixer-0.3.3\regionfixer_core\interactive.py", line 28, in <module>
        from regionfixer_core import world
    File "C:\Users\Sebastian Kuipers\Downloads\Minecraft-Region-Fixer-0.3.3\regionfixer_core\world.py", line 30, in <module>
        import nbt.region as region
    File "C:\Users\Sebastian Kuipers\Downloads\Minecraft-Region-Fixer-0.3.3\nbt\__init__.py", line 2, in <module>
        from . import *
    File "C:\Users\Sebastian Kuipers\Downloads\Minecraft-Region-Fixer-0.3.3\nbt\nbt.py", line 7, in <module>
        from collections import MutableMapping, MutableSequence, Sequence
    ImportError: cannot import name 'MutableMapping' from 'collections' (C:\Program Files\Python310\lib\collections\__init__.py)
```
**Desktop (please complete the following information):**
 - OS: windows
 - Python version: 3.10
 - Region Fixer Version v 0.3.3 (latest from releases)

**Additional context**
Please don't use deprecated functions. The deprecationwarning is there for a reason.

**Comment by Fenixin** [comment]
Thanks for the heads up.

**Comment by macfreek** [comment]
Upstream ticket: https://github.com/twoolie/NBT/issues/154

**Comment by Fenixin** [comment]
Just updated regionfixer with your branch `fix-python-3.10` and everything seems to work as expected. Thanks a lot @macfreek for your update.

@sebkuip I'm going to close this, feel free to reopen if needed.
